content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -85,11 +85,13 @@ To account for modern web applications, loading time watches for network request
   - The difference between `navigationStart` and `loadEventEnd`.
   - Or the difference between `navigationStart` and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
-- **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
+- **SPA Route Change**: Loading Time is equal to the difference between the URL change and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 ### How page activity is calculated
 
-Whenever a navigation or a click occurs, the RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+Whenever a navigation or an URL change occurs, the RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think "click" here could have been about how we compute page activity for click actions. But since this page is about *page* monitoring, it makes sense to avoid talking about click actions here. We could have a similar paragraph on the [Track User Actions](https://docs.datadoghq.com/real_user_monitoring/browser/tracking_user_actions/) doc though.

Reply:
oh, that is why click was mentioned!
What about removing `Whenever a navigation or an URL change occurs`?
Then, this paragraph would only talk about how it is calculated and not when, then it could be accurate for both contexts.
wdyt?

Reply:
Yes that could be nice!